pkg/transforms/vapbinding.go

```diff
@@ -51,6 +89,105 @@ func (v VapBindingResource) BuildNode() Node {
 
 // BuildEdges construct the edges for VapBindingResource
 func (v VapBindingResource) BuildEdges(ns NodeStore) []Edge {
-	// no op for now to implement interface
-	return []Edge{}
+	policyName, ok := v.node.Properties["policyName"].(string)
+	if !ok || policyName == "" {
+		return []Edge{}
+	}
+
+	nodeInfo := NodeInfo{
+		Name:      v.node.Properties["name"].(string),
+		NameSpace: "_NONE",
+		UID:       v.node.UID,
+		EdgeType:  "attachedTo",
+		Kind:      v.node.Properties["kind"].(string),
+	}
+
+	propSet := map[string]struct{}{policyName: {}}
+
+	edges := edgesByDestinationName(propSet, "ValidatingAdmissionPolicy", nodeInfo, ns, []string{})
+
+	if v.paramRef == nil {
+		return edges
+	}
+
+	policy, ok := ns.ByKindNamespaceName["ValidatingAdmissionPolicy"]["_NONE"][policyName]
+	if !ok {
+		return edges
+	}
+
+	paramKind, ok := policy.Properties["paramKind_kind"].(string)
```

Comment:
The [node object has a Metadata field](https://github.com/stolostron/search-collector/blob/main/pkg/transforms/transformer.go#L60-L65) that should be used to store any data needed to build the edges.
The difference is that 
- anything stored in `node.Properties` is sent to the hub and becomes a searchable field.
- data stored in `node.Metadata` is kept within the collector only, so this is used for things like building edges.

Reply:
Good point. I updated the generic transformer configuration to allow metadata only fields. Let me know what you think!